pocs/observatory.py

```diff
@@ -128,6 +135,9 @@ def status(self):
                     status['mount']['mount_target_ha'] = self.observer.target_hour_angle(
                         t, self.mount.get_target_coordinates())
 
+            if self.dome:
+                status['dome'] = self.dome.status
```

Comment:
I realize this has already been merged, but it's a little odd that `status` is a property here but a method for everything else. Why not make the dome status a method?